client/src/components/GraphViewerComponent/GraphViewerRelationshipCreateComponent/GraphViewerRelationshipCreateComponent.js

```diff
@@ -51,17 +63,38 @@ export class GraphViewerRelationshipCreateComponent extends Component {
     };
   }
 
+  btnIconStyles = {
```

Comment:
Most of our styles are in SCSS files alongside the component. Unless there is a reason to do them in code, we should probably stick to that approach.

Reply:
Ready